src/main/src/main/webapp/WEB-INF/web.xml

```diff
@@ -4,4 +4,13 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
 		 http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
          version="3.1">
+
+    <servlet>
+        <servlet-name>main</servlet-name>
+        <servlet-class>org.springframework.web.servlet.DispatcherServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>main</servlet-name>
+        <url-pattern>/</url-pattern>
```

Comment:
Please, change to /*

Reply:
done

Reply:
but got HTTP Status 404 - description The requested resource is not available.